roles/openshift_preflight/facts/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: set common variables
+  set_fact:
+    deployment_type: "{{ deployment_type | default('openshift-enterprise') }}"
```

Comment:
@rhcarvalho for testing for atomic host, it seems that you can simply test for the existence of `/run/ostree-booted`.

I'm not sure that testing for a RHEL host is sufficient for testing for Origin/OCP, though. One could install Origin on RHEL.